app/graphql/types/user_type.rb

```diff
@@ -4,5 +4,6 @@ class UserType < Types::BaseObject
     field :name, String, null: true
     field :email, String, null: true
     field :birthdate, String, null: true
+    field :events, [Types::EventType], null: false
```

Comment:
Allows you to query events from a user query. This should match Leah's era's query as well.